src/pages/ContractInfoPage/components/ContractInfo/ContractInfoHeader.vue

```diff
@@ -142,6 +149,10 @@ const iconName = computed(() =>
   line-height: toRem(32);
   font-weight: 400;
 
+  &--write {
+    color: var(--primary-main);
+  }
+
```

Comment:
`writable` / `type-write` / `action-write` would be more descriptive naming for class modificator.

Reply:
done